agent/ecscni/plugin.go

```diff
@@ -31,7 +31,11 @@ import (
 	"github.com/pkg/errors"
 )
 
-const currentCNISpec = "0.3.1"
+const (
+	currentCNISpec = "0.3.1"
+	currentCNIVersion = "2018.08.0"
+	CNIGitHash = "a134a973585b560439ed25ec3857e4789bfeb89f"
```

Comment:
We update these values each time we update the CNI plugin, yeah? 
If so, could you please add a comment about it?

Reply:
Yes, we update this every time we update the CNI plugin. Will add a comment.